proto/provisioner.proto

```diff
@@ -43,5 +44,14 @@ message DatabaseResponse {
 }
 
 message DatabaseDeletionResponse {}
+
+message QdrantRequest {
+  string project_name = 1;
+}
+
+message QdrantResponse {
+  string url = 1;
+}
```

Comment:
Should we name this in a less qdrant specific way? We may add similar resources in the future that could also use these messages.

Reply:
Yeah I'm inclined to go down this path. Would have to look at how similarly other integrations want to use the local provisioner server for docker, and see if there is an easy pattern to follow. It might be that each type of plugin is too unique for a generalized solution.